next/src/form.ts

```diff
@@ -272,42 +277,33 @@ function validate(value: SchemaValue, schema: JsfSchema, options: ValidationOpti
   return result
 }
 
-export interface ValidationOptions {
```

Comment:
Moved to `src/validation/schema.ts` to avoid circular dependency between `src/form.ts` and `src/visibility.ts`.